test/unit/sdk.test.js

```diff
@@ -819,7 +789,7 @@ describe('sdk', () => {
 
       document.getElementById(id).click();
 
-      expect(mockOpen).toHaveBeenCalledWith(expectedLink, 'Connect your car', expectedOptions);
```

Comment:
why did we remove the expectations of the call?

Reply:
Basically, it is impossible to test this exact way unless we can figure out how to mock window.screen to set width and height values. This is because it needs a primitive value, but the test environment only allows `screen.width` and height to be 0. It felt wrong to test it with incorrect values.

Instead, I came up with a new test implementation. See my updated commits. Thoughts?